lint/internal/policy/funcs/unix.go

```diff
@@ -27,15 +28,22 @@ var GrepFunc = function.New(&function.Spec{
 		pattern := args[0].AsString()
 		text := args[1].AsString()
 		var matches []string
-		in := strings.NewReader(text)
-		scanner := bufio.NewScanner(in)
```

Comment:
TIL: `bufio.Scanner` has a limitation to read maximum string 🤔 Also, `MaxScanTokenSize(64 * 1024)` is a const value!